scripts/deploy_assisted_service.sh

```diff
@@ -9,12 +9,13 @@ export NAMESPACE=${NAMESPACE:-assisted-installer}
 export SERVICE_URL=$(get_main_ip)
 export SERVICE_PORT=${SERVICE_PORT:-6000}
 export SERVICE_BASE_URL="http://${SERVICE_URL}:${SERVICE_PORT}"
+export ENABLE_AUTH=${ENABLE_AUTH:-false}
 
 mkdir -p build
 
 print_log "Updating assisted_service params"
-skipper run discovery-infra/update_assisted_service_cm.py
-skipper run "make -C assisted-service/ deploy-all" ${SKIPPER_PARAMS} DEPLOY_TAG=${DEPLOY_TAG} NAMESPACE=${NAMESPACE}
+skipper run discovery-infra/update_assisted_service_cm.py ENABLE_AUTH=${ENABLE_AUTH}
+skipper run "make -C assisted-service/ deploy-all" ${SKIPPER_PARAMS} DEPLOY_TAG=${DEPLOY_TAG} NAMESPACE=${NAMESPACE} ENABLE_AUTH=${ENABLE_AUTH}
```

Comment:
Also I'm thinking that these changes might not be needed?

Do we have a use case for deploying assisted-service inside minikube and also configuring for auth against the red hat sso? Wouldn't we just use the instance on cloud.redhat.com at that point?

If we do want this available, then I'll need another PR to add the credentials assisted-service will use to do the reverse lookup of user info in https://github.com/openshift/assisted-service/pull/48 (which most people won't have access to)